crypto/msgp_gen.go

```diff
@@ -3,6 +3,7 @@ package crypto
 // Code generated by github.com/algorand/msgp DO NOT EDIT.
 
 import (
+	"C"
 	_ "runtime/cgo"
 	_ "unsafe"
```

Comment:
Out of curiosity, are these imports required for something important, or are they just an artifact of codegen?

Reply:
I think it's just an artifact -- msgp runs `goimports` to clean up the import statements, and I don't really know what rules `goimports` applies, other than the fact that it makes sure that the compiler doesn't complain about unused or missing imports.